app/main/constants/config-const.js

```diff
@@ -6,7 +6,7 @@ angular.module('main')
   ENV: {
     /*inject-env*/
     'SERVER_URL': 'http://sample-env.qtbvngb2iz.us-west-2.elasticbeanstalk.com/',
```

Comment:
I see. in that case, we have to change the environment name in AWS. I will check that.